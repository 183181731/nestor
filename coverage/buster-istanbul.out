Running tests ...
[1A[KRunning 99 tests in 1 runtime ...
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 16% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 79% done[0m
[1A[K[1m[32m99 tests, 428 assertions, 1 runtime ... OK[0m
-----------------------|-----------|-----------|-----------|-----------|
File                   |   % Stmts |% Branches |   % Funcs |   % Lines |
-----------------------|-----------|-----------|-----------|-----------|
   lib/                |       100 |       100 |       100 |       100 |
      cli.js           |       100 |       100 |       100 |       100 |
      jenkins.js       |       100 |       100 |       100 |       100 |
   lib/api/            |       100 |       100 |       100 |       100 |
      consolestream.js |       100 |       100 |       100 |       100 |
      jenkins.js       |       100 |       100 |       100 |       100 |
      job.js           |       100 |       100 |       100 |       100 |
      util.js          |       100 |       100 |       100 |       100 |
      view.js          |       100 |       100 |       100 |       100 |
   lib/cli/            |       100 |       100 |       100 |       100 |
      jenkins.js       |       100 |       100 |       100 |       100 |
      job.js           |       100 |       100 |       100 |       100 |
      util.js          |       100 |       100 |       100 |       100 |
      view.js          |       100 |       100 |       100 |       100 |
-----------------------|-----------|-----------|-----------|-----------|
All files              |       100 |       100 |       100 |       100 |
-----------------------|-----------|-----------|-----------|-----------|


=============================== Coverage summary ===============================
Statements   : 100% ( 609/609 )
Branches     : 100% ( 102/102 )
Functions    : 100% ( 162/162 )
Lines        : 100% ( 609/609 )
================================================================================
