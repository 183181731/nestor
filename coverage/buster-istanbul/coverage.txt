-----------------------|-----------|-----------|-----------|-----------|
File                   |   % Stmts |% Branches |   % Funcs |   % Lines |
-----------------------|-----------|-----------|-----------|-----------|
   lib/                |       100 |       100 |       100 |       100 |
      cli.js           |       100 |       100 |       100 |       100 |
      jenkins.js       |       100 |       100 |       100 |       100 |
   lib/api/            |       100 |       100 |       100 |       100 |
      consolestream.js |       100 |       100 |       100 |       100 |
      jenkins.js       |       100 |       100 |       100 |       100 |
      job.js           |       100 |       100 |       100 |       100 |
      util.js          |       100 |       100 |       100 |       100 |
      view.js          |       100 |       100 |       100 |       100 |
   lib/cli/            |       100 |       100 |       100 |       100 |
      jenkins.js       |       100 |       100 |       100 |       100 |
      job.js           |       100 |       100 |       100 |       100 |
      util.js          |       100 |       100 |       100 |       100 |
      view.js          |       100 |       100 |       100 |       100 |
-----------------------|-----------|-----------|-----------|-----------|
All files              |       100 |       100 |       100 |       100 |
-----------------------|-----------|-----------|-----------|-----------|
