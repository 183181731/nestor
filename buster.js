module.exports.unit = {
  environment: 'node',
  sources: [
    'lib/**/*.js'
  ],
  tests: [
    'test/**/*.js'
  ],
  extensions: [
    require('buster-istanbul')
  ],
  'buster-istanbul': {
    outputDirectory: '.bob/report/coverage'
  }
};
