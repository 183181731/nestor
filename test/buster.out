Running tests ...
[1A[KRunning 99 tests in 1 runtime ...
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32mRunning 99 tests in 1 runtime ... 62% done[0m
[1A[K[1m[32m99 tests, 428 assertions, 1 runtime ... OK[0m
